frontend/src/containers/pages/InvalidRolePage.jsx

```diff
@@ -22,27 +22,32 @@ function InvalidRolePage({ group }) {
     fontWeight: "600",
     display: "flex",
     flexDirection: "column",
+    height: "100vh",
```

Comment:
**Not an issue**, but on mobile, it's usually advised to consider using `dvh` **if** this was in any way targetted for mobile.